internal/app/components/plans/plans.go

```diff
@@ -9,14 +9,19 @@ type Plan interface {
 	GetName() string
 }
 type Plans []Plan
+type PlanId string
```

Comment:
Note: NewTimeline func in timeline.go needs to be changed so it copies plans instead of initializing an empty plan

Reply:
I think a new timeline is expected to have empty plans